metis-enrichment/metis-enrichment-rest/src/main/java/eu/europeana/enrichment/rest/config/Application.java

```diff
@@ -56,6 +56,8 @@
   @Value("${enrichment.mongo.database}")
   private String enrichmentMongoDatabase;
 
+  @Value("${mongo.application.name}")
```

Comment:
Maybe follow the naming pattern here(with the enrichment prefix)?

Reply:
Good point, I'm going to take care of it